src/Routes.jsx

```diff
@@ -50,12 +49,6 @@ const Routes = (props) => (
     <Route exact path="/home" render={(routeProps) => <Home {...routeProps} {...props} />} />
     <Route exact path="/status" render={(routeProps) => <Status {...routeProps} {...props} />} />
     <Route exact path="/liveness" component={HealthCheck} />
-    <Route exact path="/backgroundsignin" render={
-      (routeProps) =>
-        checkEnv(envGroups.NON_STAGING)
-          ? <BackgroundSignIn {...routeProps} />
-          : <NotFound />
-    } />
```

Comment:
This page is what we use for E2E tests so we didn't have to manually trigger an auth dance. Currently, they're not run on PRs or in GHA, but we can run it locally. We either need a B2C version of this, or bring it back so we can test authenticated pages.